web/client/selectors/geostory.js

```diff
@@ -287,3 +287,9 @@ export const hasPendingChanges = (state = {}) => state?.geostory?.pendingChanges
  * @param {object} state application state
  */
 export const updateUrlOnScrollSelector = state => get(state, 'geostory.updateUrlOnScroll', false);
+
+/**
+ * Gets the currentStoryFonts that were loaded from config
+ * @param {object} state application state
+ */
+export const currentStoryFonts = state => get(state, "geostory.currentStory.settings.fontFamilies", []);
```

Comment:
please ass a unit test for this selector

in general selectors, actions, reducers, utils should be fully tested and mostly covered also because they are easier to test